src/app/ReadHandler.cpp

```diff
@@ -62,7 +62,7 @@ CHIP_ERROR ReadHandler::Init(Messaging::ExchangeManager * apExchangeMgr, Interac
     mIsChunkedReport    = false;
     mInteractionType    = aInteractionType;
     mInitiatorNodeId    = apExchangeContext->GetSessionHandle().GetPeerNodeId();
-    mFabricIndex        = apExchangeContext->GetSessionHandle().GetFabricIndex();
+    mSubjectDescriptor  = apExchangeContext->GetSessionHandle().GetSubjectDescriptor();
```

Comment:
How big will this data structure end up being?

Reply:
It's a fabric index, an auth mode, (together 2 bytes)
a node ID (8 bytes),
and a few more values (CATs, probably 3x4 bytes).
We can likely change the "Peer Node ID" to just be the subject node ID in the subject descriptor.